pennylane/measurements/expval.py

```diff
@@ -20,11 +20,15 @@
 import pennylane as qml
 from pennylane.operation import Operator
 from pennylane.wires import Wires
+
 from .measurements import Expectation, SampleMeasurement, StateMeasurement
+from .sample import SampleMP
 from .mid_measure import MeasurementValue
 
 
-def expval(op: Union[Operator, MeasurementValue]):
+def expval(
+    op: Union[Operator, MeasurementValue] = None,
```

Comment:
```suggestion
    op: Union[Operator, MeasurementValue],
```
Should probably keep having this be mandatory.

Reply:
Ooops, forgot about this one.